### PR TITLE
Fix integration and pack metadatas categories

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fix integration and pack metadatas categories to match the new standards. 

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
Happy with the category changes made. Letting ENG to approve.

File: Packs/CyCognito/pack_metadata.json [hunk 1]
```diff
@@ -7,9 +7,7 @@
     "url": "",
     "email": "support@cycognito.com",
     "categories": [
-        "Vulnerability Management",
-        "Data Enrichment & Threat Intelligence",
-        "External Exposure"
+        "Data Enrichment & Threat Intelligence"
```
Comment (JudahSchwartz) [inline Packs/CyCognito/pack_metadata.json:10]: we dont need releaese notes for this pack?

File: Packs/CaseManagement-Generic/pack_metadata.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "url": "",
     "email": "",
     "created": "2020-08-20T14:37:35Z",
-    "categories": [],
+    "categories": ["Case Management"],
     "tags": [],
```
Comment (JudahSchwartz) [inline Packs/CaseManagement-Generic/pack_metadata.json:11]: need release notes

**Review by JudahSchwartz — APPROVED** [review]
looks good to me as long as passes validation. Might be a good idea to split up if we cant get this in in 1 pr

File: Packs/CyCognito/pack_metadata.json (hunk 1 above)
Comment (YuvHayun) [reply Packs/CyCognito/pack_metadata.json:10]: No need for RN when changing pack_metadata.
I ran `update-release-notes -g` to ensure everything gets updated.

**Pull request merged**